modules/tensor_mechanics/test/tests/scalar_material_damage/ad_nonlocal_scalar_damage.i

```diff
@@ -63,7 +63,7 @@
     material_name = local_damage_reg
     execute_on = "INITIAL timestep_end"
     block = 0
-    radius = 0.3
+    radius = 0.55
```

Comment:
Curious about why the radius was changed. Is this a more demanding test now?

Reply:
See above. Prior to this PR the value was the square root of the radius